content/blog/v1-35-released.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Scratch Addons v1.35 is here!
+description: Skew shapes in costumes, make text larger, and more!
+date: 2023-10-11T00:00:00Z
+author: DNin01
+---
+
+It's time for another update, filled with exciting tools for project creation and some accessibility improvements!
+
+### An all-new costume editing feature
+
+Our first new addon is a costume editor feature - it's called "Skewing in costume editor".
+
+By skewing, we mean this:
+
+![Skewing demonstration](/assets/img/blog/v1-35-released/skew-demo.gif)
+
+You can use this to create slanted objects and italic text! To skew an object, enable the addon, then, while dragging a horizontal or vertical resize handle, hold <kbd>Ctrl</kbd> and drag parallel to the side. Hold <kbd>Ctrl</kbd> and <kbd>Alt</kbd> together to skew it out in both directions.
+
+And yes, you can skew a selection of multiple objects as well.
+
+### This feature was hidden for some random reason
+
+Did you know that `point towards` blocks accept "random direction" as an option? That's right, we turned this into an addon so you can do that. It just saves a `pick random` block.
+
+!["point towards mouse-pointer" block with dropdown open and an option for random direction highlighted](/assets/img/blog/v1-35-released/random-dir.png)
+
+### Block text now comes in XL
+
+Not only is color contrast important for readability, but also size. That's why you will now be able to increase the size of the text on blocks through the "Customizable block text style" addon.
+
+![Block palette with larger text on blocks](/assets/img/blog/v1-35-released/large-text.png)
+
+The nice thing about this is it also affects the block palette.
+
+### Choose your own increments
+
+The "Arrow key incrementation in editor" addon has been updated and now supports setting custom increment values. Wanted to increment by more precise values like 0.25, 5, or 100? Now you can, just flick the switch "use custom values" and type the values into the input field!
+
+Please note that the numbers you enter can only get so significant - the fields don't accept super precise values like `0.1234` or extremely large numbers like `10000`. This change just helps you reach in-betweens that weren't before possible with the multi-choice setting.
+
+The addon is also now compatible with some other input fields, such as the sprite properties inputs, so now those can be incremented with the arrow keys too. Lovely!
+
+### See how your description looks to others
+
+If you can't click links on editable fields, how are you supposed to know whether a link, mention, or hashtag in your project's Instructions or Notes and Credits section can actually be followed? Well, the new "Preview project instructions and notes" addon allows you to preview what your project page looks like to other users.
```

Comment:
It would probably be good to mention that this might have some issues, something like this:

```suggestion
If you can't click links on editable fields, how are you supposed to know whether a link, mention, or hashtag in your project's Instructions or Notes and Credits section can actually be followed? Well, the new "Preview project instructions and notes" addon allows you to preview what your project page looks like to other users. Note that this addon is currently in beta, and you may experience some issues. Feel free to [leave feedback](/feedback) if you do!
```